content/SpeedGauge.ui.qml

```diff
@@ -0,0 +1,346 @@
+/*
+This is a UI file (.ui.qml) that is intended to be edited in Qt Design Studio only.
+It is supposed to be strictly declarative and only uses a subset of QML. If you edit
+this file manually, you might introduce QML code that is not supported by Qt Design Studio.
+Check out https://doc.qt.io/qtcreator/creator-quick-ui-forms.html for details on .ui.qml files.
+*/
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Studio.Components 1.0
+import QtQuick.Shapes 1.0
+import QtQuick.Studio.Effects 1.0
+import QtQuick.Timeline 1.0
+import QtQuick.Studio.LogicHelper 1.0
+
+Item {
+    id: speed
+    width: 500
+    height: 500
+    //property alias int_kmh
+
+    Image {
+        id: main
+        x: 0
+        y: 0
+        width: 500
+        height: 500
+        source: "SpeedGaugePicture/MainBackground.svg"
+        scale: 1
+        fillMode: Image.PreserveAspectFit
+
+        Text {
```

Comment:
Maybe it's worth to use a RowLayout? https://doc.qt.io/qt-6/qtquicklayouts-index.html